CHANGELOG.md

```diff
@@ -1,5 +1,33 @@
 # Version changelog
 
+## [Release] Release v0.230.0
+
+Notable changes for Databricks Asset Bundles:
+
+Workspace paths are automatically prefixed with `/Workspace`. In addition, all usage of path strings such as `/Workspace/${workspace.root_path}/...` in bundle configuration is automatically replaced with `${workspace.root_path}/...` and generates a warning as part of bundle validate.
+
+More details can be find here: https://docs.databricks.com/en/release-notes/dev-tools/bundles.html#workspace-paths-will-be-automatically-prefixed
```

Comment:
```suggestion
More details can be found here: https://docs.databricks.com/en/release-notes/dev-tools/bundles.html#workspace-paths-will-be-automatically-prefixed
```

Reply:
And the link should be the same as in the warning: https://docs.databricks.com/en/release-notes/dev-tools/bundles.html#workspace-paths -- I suspect this anchor to be stable where the one linked now isn't (it's the same as the title and titles are automatically anchored).